bootstrap/src/lib/cache-warmer/config.ts

```diff
@@ -0,0 +1,35 @@
+import objectHash from 'object-hash'
+
+export interface Config {
+  /**
+   * The interval in milliseconds which the warm-up engine will execute
+   * the underlying external adapter to update its cache
+   */
+  warmupInterval: number
```

Comment:
I think its best to keep it a little more descriptive since in practice it's confusing to just see `interval`, especially when using destructured objects

Reply:
Both styles make sense to me in different contexts. Naming is hard, and this is a simple example lol.

A non-issue at this point!